microservices-swift/Sources/Application/Routes/BoardgamesRoutes.swift

```diff
@@ -1,37 +1,41 @@
-import LoggerAPI
-import Health
+import Kitura
 import KituraContracts
 
-func initializeBoardgamesRoutes(app: App) {
-    let seed = [Boardgame(id: 1, name: "Risk", emoji: "💣"), Boardgame(id: 2, name: "Uno", emoji: "1️⃣")]
-    var boardgames: [Boardgame] = seed
+class BoardgameRoutes {
+    private static let seed = [Boardgame(id: 1, name: "Risk", emoji: "💣"), Boardgame(id: 2, name: "Uno", emoji: "1️⃣")]
```

Comment:
Not sure because swift but shouldn't `seed` be `SEED`?

Reply:
Conventions in Swift are a bit different than JAVA/Kotlin, constants are still named using lowercase characters

Reply:
Good to know, thanks 👍